ldapadmin/src/main/filtered-resources/WEB-INF/views/privateUi.jsp

```diff
@@ -0,0 +1,111 @@
+<%@ page contentType="text/html; charset=UTF-8" %>
+<%@ page pageEncoding="UTF-8" %>
+<%@ page isELIgnored="false" %>
+<%@ page import="org.springframework.context.ApplicationContext" %>
+<%@ page import="org.springframework.web.servlet.support.RequestContextUtils" %>
+<%
+// the context path (might not be the public context path ! -> to be improved with https://github.com/georchestra/georchestra/issues/227)
+String context = request.getContextPath().split("-")[0]; // eg /ldapadmin
+%>
+<!DOCTYPE html>
+<html lang="en" ng-app="ldapadmin">
+  <head>
+    <meta charset="UTF-8">
+    <title>LDAPadmin - geOrchestra</title> <!-- TODO: dynamic instance name -->
+    <link rel="stylesheet" href="lib/bootstrap/css/bootstrap.min.css" />
+
+    <link rel="stylesheet" href="css/main.css" />
```

Comment:
main.css does not exist anymore, since it has been removed by this commit. We should remove the line because it generates an extra hit which ends up with a 404 error.